.github/workflows/plugins_react_tests.yml

```diff
@@ -0,0 +1,38 @@
+name: (non-blocking) React on plugins
```

Comment:
The only thing I would suggest changing is naming this `Katello React tests (non-blocking` or something more specific to Katello. It's not a generalized test yet and is specific to Katello, so lets just call it that for clarity.

Reply:
I'd agree. If we are testing just katello so far, let's call it Katello tests

Reply:
I think it makes sense to leave it as is, as now when they are properly set under the same folder, any other plugin can add itself exactly like `Katello` does it, so I would expect this file to grow, e.g:

```yml
      - name: Checkout PLUGIN_NAME
        uses: actions/checkout@v2
        with:
          repository: PLUGIN_REPO/PLUGIN_NAME
          ref: master
          path: ./projects/PLUGIN_NAME
      - name: Install PLUGIN_NAME npm dependencies
        run: npm i
        working-directory: ${{ github.workspace }}/projects/PLUGIN_NAME
      - name: Run PLUGIN_NAME tests
        run: npm test
        working-directory: ${{ github.workspace }}/projects/PLUGIN_NAME
```

Reply:
@laviro The name is a pretty easy thing to update once that happens. We also aren't sure if it makes more sense to make another action to run in parallel if another plugin JS test suite is introduced, rather than including in this action.

I generally don't like to make decisions based on assumptions on what will happen to a codebase in the future, I've seen things not go as predicted one too many times and a lot of unnecessary cruft introduced and left to deal with because "something will happen" and it doesn't. So it's a bit of a pet peeve of mine :disappointed: 

I don't consider it a blocker or anything, you have my ACK, but just want to explain why I suggested that :)

Reply:
@laviro could you add those plugins right away? :) let say the plugins that are upfront with react adoption like tasks maybe ansible? :)

Reply:
@johnpmitsch Thanks for the explanation !
@ezr-ondrej yes, added the `foreman-tasks` and `foreman_rh_cloud` plugins
as I'm planning to direct them soon to use foreman code instead of the test mocks